src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -828,6 +829,24 @@ public boolean isRejectDefendantPaymentPlanNo() {
             || !paymentPlan.contains(getDefenceAdmitPartPaymentTimeRouteRequired());
     }
 
+    @JsonIgnore
+    public boolean isSettlePartAdmitClaimYes() {
+        return (
+            getApplicant1ProceedsWithClaimSpec() == null
+                && PART_ADMISSION.equals(getRespondent1ClaimResponseTypeForSpec())
+                && YesOrNo.YES.equals(getApplicant1PartAdmitIntentionToSettleClaimSpec())
+                && YesOrNo.YES.equals(getApplicant1PartAdmitConfirmAmountPaidSpec()));
+    }
+
+    @JsonIgnore
+    public boolean isSettlePartAdmitClaimNo() {
+        return (
+            getApplicant1ProceedsWithClaimSpec() != null
+                || !PART_ADMISSION.equals(getRespondent1ClaimResponseTypeForSpec())
+                || YesOrNo.NO.equals(getApplicant1PartAdmitIntentionToSettleClaimSpec())
```

Comment:
same here to avoid code repetition and also to be able to test in the case data test this condition